config/kafka-instance-types-configuration.yaml

```diff
@@ -0,0 +1,47 @@
+# This configuration defines a list of supported Kafka instance types and their available sizes for KAS Fleet Manager. 
+# Modify this file to add additional Kafka instance types and sizes or change the default values listed below.
+# 
+# The following properties must be defined for each Kafka instance type:
+#   - id: Identifier for the Kafka instance type. Each instance type name should be unique.
+#   - sizes: A list of sizes available for this instance type (should not be an empty list)
+#
+# The following properties must be defined for each size (all values must be larger than '0'):
+#   - id: The size identifier. Each size id should be unique.
+#   - ingressThroughputPerSec: Maximum limit for ingress throughput per second (MB/s).
+#   - egressThroughputPerSec: Maximum limit for egress throughput per second (MB/s).
+#   - totalMaxConnections: Maximum client connections allowed.
+#   - maxConnectionAttemptsPerSec: Maximum client connection attempts per second.
+#   - maxDataRetentionSize: Maximum storage (GB).
+#   - maxDataRetentionPeriod: Maximum duration(ISO8601) for retaining data.
+#   - maxPartitions: Maximum partitions for all topics combined.
+#   - quotaConsumed: Quota consumed for selecting this size for a Kafka instance.
+#   - quotaType: Quota type that will be consumed when this size is selected.
+#   - capacityConsumed: Data plane cluster capacity consumed by this Kafka instance size (only used for manual scaling)
+---
+supported_instance_types:
+  - id: standard
+    sizes:
+    - id: x1
+      ingressThroughputPerSec: "50Mi"
```

Comment:
It would be better if we can have separate value and unit so that UI can handle translations and custom unit if required. For UI point of view we are not going to use backend units, we are using custom units which defined in UI translation files which will helpful for translate units.
 
For now UI showing as given below
```
Ingress
up to 30 MiB/second each

Egress
up to 30 MiB/second each
```
Proposal is separate value and unit 
`ingressThroughputPerSec: {value:50, unit:"Mi"}`

Reply:
@ajaypratap003 is that proposal documented somewhere? In ADR-81 there are actually no units on the response: https://github.com/bf2fc6cc711aee1a0c2a/architecture/pull/42/files#diff-4a09f9937fd7e06026f7cfe0e7d4143d202388410486747c30750de03aca4a26R125

Reply:
ping @wtrocki

Reply:
@pb82 this proposal is from my side for UI point of view. Even UI don't need units, need value only. Avoid units change.